helm-charts/seldon-core-operator/templates/deployment_seldon-controller-manager.yaml

```diff
@@ -104,11 +104,11 @@ spec:
           protocol: TCP
         resources:
           limits:
-            cpu: {{ .Values.manager.cpuLimit }}
-            memory: {{ .Values.manager.memoryLimit }}
+            cpu: '{{ .Values.manager.cpulimit }}'
```

Comment:
It seems that this change should also be made in the values.yaml otherwise it will think it's blank. 

It seems even with this I'm getting the error: ```Error: Deployment in version "v1" cannot be handled as a Deployment: v1.Deployment.Spec: v1.DeploymentSpec.Template: v1.PodTemplateSpec.Spec: v1.PodSpec.Containers: []v1.Container: v1.Container.Resources: v1.ResourceRequirements.Limits: unmarshalerDecoder: quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$', error found in #10 byte of ...|:{"cpu":"","memory":|..., bigger context ...|"protocol":"TCP"}],"resources":{"limits":{"cpu":"","memory":"300Mi"},"requests":{"cpu":"100m","memor|...```

Reply:
Is there a reason why this is being changed to lowercase tho? Seems like a typo

Reply:
Done